extensions/amp-story/1.0/pagination-buttons.css

```diff
@@ -105,15 +105,9 @@ amp-story.i-amphtml-story-desktop-one-panel .i-amphtml-story-button-move,
   filter: none !important;
   background-color: white !important;
   border-radius: 50% !important;
-  width: 40px !important;
-  height: 40px !important;
-  background-position-y: 52% !important;
-  background-size: 20% !important;
-}
-
-amp-story.i-amphtml-story-desktop-one-panel .i-amphtml-story-fwd-replay > .i-amphtml-story-button-move {
-  background-size: 52% !important;
-  background-position-y: 56% !important;
+  width: 48px !important;
+  height: 48px !important;
```

Comment:
Could we use 100% here? Or maybe dimensions aren't needed since it's the size of the SVG 🤔

Reply:
I'll try that and get back, but definitely would be cool to get rid of these.